File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int , int > psf ; 
        for(int i = 0 ; i < nums.size() ; i++)
        {
            int k = nums[i] ; 
            int rem = target - k ; 
            if(psf.find(rem) != psf.end())
            {
                return {psf[rem] , i };
            }
            psf[k] = i ; 
        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        psm = {}
        for i , n in enumerate (nums):
            diff = target - n 
            if diff in psm:
                return[psm[diff],i]
            psm[n] = i    

        